packages/mc-scripts/package.json

```diff
@@ -14,8 +14,20 @@
   "publishConfig": {
     "access": "public"
   },
-  "main": "./build/index.js",
-  "files": ["build", "config", "package.json", "LICENSE", "README.md"],
+  "main": "dist/commercetools-frontend-mc-scripts.cjs.js",
+  "module": "dist/commercetools-frontend-mc-scripts.esm.js",
+  "preconstruct": {
+    "entrypoints": ["./index.ts", "./application-runtime.ts"]
+  },
```

Comment:
The `application-runtime` is a separate entry point so that the file can be referenced from the web pack config.